migrations/20200827163426-upload-app-details.js

```diff
@@ -0,0 +1,99 @@
+
+    let appDetails = [
+      {
+        name: "samiksha",
+        displayName: "Samiksha",
+        appDescrption: "Get the app to discover more",
+        appLogo: "samiksha.png",
+        playstoreLink: "https://play.google.com/store/apps/details?id=org.shikshalokam.samiksha",
+        appStoreLink: "https://apps.apple.com/in/app/shikshalokam-samiksha/id1442066610",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        createdBy: "SYSTEM",
+        updatedBy: "SYSTEM",
+        status: "active",
+        isDeleted: false
+      },
+      {
+        name: "darpan",
+        displayName: "Darpan",
+        appDescrption: "Get the app to discover more",
+        appLogo: "darpan.jpg",
+        playstoreLink: "https://play.google.com/store/apps/details?id=org.shikshalokam.samiksha.darpan",
+        appStoreLink: "https://apps.apple.com/us/app/darpan/id1480404451",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        createdBy: "SYSTEM",
+        updatedBy: "SYSTEM",
+        status: "active",
+        isDeleted: false
+      },
+      {
+        name: "sdi",
+        displayName: "SDI",
+        appDescrption: "Get the app to discover more",
+        appLogo: "sdi.png",
+        playstoreLink: "https://play.google.com/store/apps/details?id=org.shikshalokam.samiksha.dcpcr",
+        appStoreLink: "https://apps.apple.com/in/app/school-development-index/id1444625872",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        createdBy: "SYSTEM",
+        updatedBy: "SYSTEM",
+        status: "active",
+        isDeleted: false
+      },
+      {
+        name: "pallithalam",
+        displayName: "Palli Thalam",
+        appDescrption: "Get the app to discover more",
+        appLogo: "pallithalam.png",
+        playstoreLink: "https://play.google.com/store/apps/details?id=org.shikshalokam.samiksha.palliThalam",
+        appStoreLink: "",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        createdBy: "SYSTEM",
+        updatedBy: "SYSTEM",
+        status: "active",
+        isDeleted: false
+      },
+      {
+        name: "ecekar",
+        displayName: "ECE-KAR",
+        appDescrption: "Get the app to discover more",
+        appLogo: "ecekar.png",
+        playstoreLink: "https://play.google.com/store/apps/details?id=org.shikshalokam.samiksha.kps",
+        appStoreLink: "",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        createdBy: "SYSTEM",
+        updatedBy: "SYSTEM",
+        status: "active",
+        isDeleted: false
+      }
+    ]
+
+    for( let app = 0 ; app < appDetails.length; app++ ) {
+      await kendraServiceHelper.upload(
+        `public/assets/apps/${appDetails[app].appLogo}`,
+        `static/apps/${appDetails[app].appLogo}`
+      );
+
+      appDetails[app].appLogo = "static/apps/" + appDetails[app].appLogo; 
+    }
+
+    await db.collection('apps').insertMany(appDetails);
```

Comment:
@deepakr123 - Use create API of apps in Kendra to create an app.

Reply:
resolved